test/e2e/data/infrastructure-vsphere/main/finalizers/delete-cluster-secret.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
these can get removed right?

Reply:
yes this will also go away, it was just kept for reference. If you think the combined tests should be ok, I can push a squashed commit

Reply:
yes, please drop them 👍

Reply:
(and feel free to squash)